src/github/pullRequestOverview.ts

```diff
@@ -268,7 +268,10 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel {
 
 	private async addReviewers(message: IRequestMessage<void>): Promise<void> {
 		try {
-			const allAssignableUsers = await this._folderRepositoryManager.getAssignableUsers();
+			const allAssignableUsers = await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: 'Getting all possible reviewers' },
```

Comment:
it seems a bit strange for this to pop up when getting the assignable users is really fast, like when they're already cached, should we maybe only show it then? or would passing a promise to `showQuickPick